docs/api/api/api.txt

```diff
@@ -1933,12 +1933,18 @@ XmlResult: required_checks
 == Staging
   WARNING: This API is currently in beta and unstable
 
-=== Staging Projects
+=== Staging Workflow
 
+<staging_workflow_project>: The project for which we want to create the staging workflow
```

Comment:
I think mentioning the variable once will do

Reply:
what do you mean?

Reply:
zoom out!

```
<staging_workflow_project>: The project for which we want to create the staging workflow
<staging_workflow_project>: The name of the staging workflow project, e.g. openSUSE:Factory
```